addons/mail/models/discuss/discuss_channel.py

```diff
@@ -710,13 +710,21 @@ def _message_compute_parent_id(self, parent_id):
              ('res_id', '=', self.id)
             ]).id
 
+    def _get_allowed_message_post_params(self):
+        params = super()._get_allowed_message_post_params()
+        params.add("special_mentions")
+        return params
```

Comment:
shorter using set operator
```suggestion
        return super()._get_allowed_message_post_params() | "special_mentions"
```